### PR TITLE
Refactor NodesUtilizaton for initial sync 

### LINKED ISSUE #282: Refactor `NodesUtilizaton` for initial sync
In [PR #265](https://github.com/G-Research/yunikorn-history-server/pull/265), new mechanisms for syncing `yunikorn` objects from the event stream have been implemented. 
Although `NodesUtilization` does not emit events, we can still refactor the initial sync process to integrate it with the new mechanism.

### PR DESCRIPTION
**Pull request by sudiptob2**
closes #282

based on #271

### CODE REVIEW

File: internal/webservice/routes.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func (ws *WebService) init(ctx context.Context) {
 	)
 	service.Route(
 		service.GET(routeNodeUtilization).
-			To(ws.getNodeUtilizations).
+			To(ws.getNodesUtils).
 			Produces(restful.MIME_JSON).
 			Writes([]dao.PartitionNodesUtilDAOInfo{}).
```
Comment (nikola-jokic) [inline internal/webservice/routes.go:131]: You probably need to update this as well for documentation purposes. It writes `model.NodeUtil` now

**Review by nikola-jokic — CHANGES_REQUESTED** [review]
Looks good! Minor change requested and we can merge this

**Review by nikola-jokic — APPROVED** [review]
lgtm

**Comment by sudiptob2** [comment]
Dropped node utilization in #305

**Pull request closed without merging**